### PR TITLE
Update README.md to correct grammar and improve formatting

### PR DESCRIPTION
**Pull request by CodeMaster7000**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # Chrome Extensions samples
 
 Official samples for Chrome Extensions and the Chrome Apps platform.
-Note that Chrome Apps are deprecated—learn more [on the Chromium blog](https://blog.chromium.org/2020/08/changes-to-chrome-app-support-timeline.html).
+Note that Chrome Apps are deprecated - learn more [on the Chromium blog](https://blog.chromium.org/2020/08/changes-to-chrome-app-support-timeline.html).
```
Comment (jpmedley) [inline README.md:4]:
```suggestion
Note that Chrome Apps are deprecated. Learn more [on the Chromium blog](https://blog.chromium.org/2020/08/changes-to-chrome-app-support-timeline.html).
```

Comment (jpmedley) [reply README.md:4]: The existing punctuation was essentially a 'comma' splice.

File: README.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ The directory structure is as follows:
 * [apps/](apps/) - deprecated Chrome Apps platform (not listed below)
 * [mv2-archive/](mv2-archive/) - resources for manifest version 2
 
-To experiment with these samples, please clone this repo and use 'Load Unpacked Extension'.
+To experiment with these samples, please clone this repo and use `Load Unpacked Extension`.
```
Comment (jpmedley) [inline README.md:21]: Why backticks?

**Review by jpmedley — CHANGES_REQUESTED** [review]

**Comment by CodeMaster7000** [comment]
@jpmedley I have corrected the file.

**Review by jpmedley — APPROVED** [review]

**Comment by jpmedley** [comment]
Thank you for your help.

**Pull request merged**